packages/govuk-frontend/src/govuk/components/skip-link/skip-link.mjs

```diff
@@ -10,18 +10,17 @@ export class SkipLink extends GOVUKFrontendComponent {
   /** @private */
   $module
 
-  /**
-   * @private
-   * @type {HTMLElement | null}
-   */
-  $linkedElement = null
+  /** @private */
+  $linkedElement
 
   /** @private */
   linkedElementListener = false
 
   /**
    * @param {Element} $module - HTML element to use for skip link
-   * @throws {MissingElementError} If the element with the specified ID is not found
+   * @throws {ElementError} when $module is not set or the wrong type
+   * @throws {ElementError} when $module.hash does not contain a hash
+   * @throws {ElementError} when the linked element is missing or the wrong type
```

Comment:
@romaricpascal Just flagging that @domoscargin has documented `@throw` for [JSDoc output](https://govuk-frontend-review.herokuapp.com/docs/javascript/classes/SkipLink.html#constructor)

We only need them on public methods but I've listed them based on the test names